pyproject.toml

```diff
@@ -60,4 +61,5 @@ requires = ["poetry-core"]
 build-backend = "poetry.core.masonry.api"
 
 [tool.isort]
-profile = "black"
\ No newline at end of file
+profile = "black"
+skip = ["scripts/metadata/load_open_metadata_backend.py"]
```

Comment:
Skipping this here because the open-metadata SDK structure doesn't lend itself to clear imports so I find it better to organize the package imports logically